### DISCUSSION TITLE
Can we have the Twin validation to use with any framework?

### DISCUSSION THREAD

**Discussion by brunoti** [ben-rogerson/twin.macro#865] [General]
I think one of the best features here is the validation that avoids us to make typos and dig into multiple classes to see where we did it wrong.

Is it possible to use just that part of the project in other places?

**Comment by ben-rogerson** [comment]
Glad you like that feature :)

Currently twin's suggestion feature is only available in twin once it scans over your classes and doesn't find any matches.

I don't wish to inspire too much hope but there's perhaps room for a port some of twins features over to VSCode plugins. There's a bit of work involved but I'll have a look into it 👍
